lib/appsignal/config.ex

```diff
@@ -104,11 +104,15 @@ defmodule Appsignal.Config do
   """
   @spec valid?() :: boolean
   def valid? do
-    Application.get_env(:appsignal, :config)[:push_api_key]
-    |> empty?
-    |> Kernel.not()
+    do_valid?(Application.get_env(:appsignal, :config)[:push_api_key])
   end
 
+  defp do_valid?(push_api_key) when is_binary(push_api_key) do
+    String.trim(push_api_key) != ""
```

Comment:
Maybe the length check is better here using `empty?`?

Reply:
Done!